packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/widgets.tsx

```diff
@@ -76,6 +87,7 @@ const RadioGroupOptionsWidget = ({ displayed_trade_param, modal_title }) => {
 
     return (
         <React.Fragment>
+            {/* @ts-expect-error TODO: check if TS error is gone after RadioGroupOptionsModal is migrated to TS */}
```

Comment:
Please help to create a tech debt card for `RadioGroupOptionsModal`.
https://app.clickup.com/20696747/v/l/li/900502075346

Reply:
no need. its in this card/PR that im pushing to get merged

https://app.clickup.com/t/20696747/OPT-149